src/Controller/StartController.php

```diff
@@ -155,11 +173,27 @@ private function createVoucher(Request $request, User $user)
     private function createRandomAlias(Request $request, User $user)
     {
         try {
-            if ($this->aliasHandler->create($user, null) instanceof Alias) {
+            if ($this->aliasHandler->create($user) instanceof Alias) {
                 $request->getSession()->getFlashBag()->add('success', 'flashes.random-alias-creation-successful');
             }
         } catch (ValidationException $e) {
-            // Should not thrown
+            $request->getSession()->getFlashBag()->add('error', $e->getMessage());
+        }
+    }
+
+    /**
+     * @param Request $request
+     * @param User    $user
+     * @param string  $alias
+     */
+    private function createCustomAlias(Request $request, User $user, string $alias)
+    {
+        try {
+            if ($this->aliasHandler->create($user, strtolower($alias)) instanceof Alias) {
```

Comment:
We should move the logic `strtolower` to the Creator to ensure that the usage of the creator is safe for any string you give as parameter.